Cargo.toml

```diff
@@ -38,6 +38,7 @@ ansi_term = "^0.12.1"
 ansi_colours = "^1.0"
 console = "0.14.1"
 lazy_static = { version = "1.4", optional = true }
+lazycell = "1.0"
```

Comment:
Note that syntect already depends on this crate, so it is not a big deal for us to also start doing that.